compute_tools/src/config.rs

```diff
@@ -75,7 +75,7 @@ pub fn write_postgres_conf(
         neon_safekeepers_value.push_str(&spec.safekeeper_connstrings.join(","));
         writeln!(
             file,
-            "neon.safekeepers={}",
+            "neon.safekeeper_connstrings={}",
```

Comment:
Also, do we want to write the new configuration format already, or have a first PR that only knows how to read the new format?